state/postgres_server_stats.go

```diff
@@ -1,7 +1,42 @@
 package state
 
+import (
+	"database/sql"
+)
+
 // PostgresServerStats - Statistics for a Postgres server.
 type PostgresServerStats struct {
 	CurrentXactId   Xid8
 	NextMultiXactId Xid8
+
+	XminHorizonBackend                sql.NullInt32
```

Comment:
[note for reviewer] I was trying to use `Xid` (or `Xid8`) here, but often these xmin horizon can be null. When it's null, I could use `COALESCE`, though I wasn't really able to create `0` xid (looks like it's hard to return an artificial transaction ID in the query), so I decided to take care of that in golang side.

Reply:
Can you clarify what issue you ran into with not being able to create `0` xid on the Postgres side?

For other's context that read this, its safe to turn NULL into 0, since that's explicitly defined as being an [invalid transaction ID](https://github.com/postgres/postgres/blob/master/src/include/access/transam.h#L31).

Reply:
I was trying to find out how to cast integer to xid or generating an artificial xid and I wasn't able to find how to do, but I was clearly not searching well.
While I was working on something else today, I found this technique, which I can totally use for this. I just needed to use text instead of int 🙃 I will update PR to use this and COALESCE 👍 

```sql
pgaweb=# select 0::xid;
ERROR:  cannot cast type integer to xid
LINE 1: select 0::xid;
                ^
pgaweb=# select '0'::xid;
 xid 
-----
   0
(1 row)
```